DoxygenRule.cmake

```diff
@@ -186,11 +186,17 @@ if(GIT_DOCUMENTATION_REPO)
       COMMENT "Copying ${PROJECT_NAME} API documentation to ${GIT_DOCUMENTATION_DIR}"
       DEPENDS ${PROJECT_NAME}-doxygen VERBATIM)
   else()
+    # Having a command while force this target to run always when invoked. That
```

Comment:
I'm becoming dyslexic.. the more I write in English the more I made this word substitution mistakes.